src/assets/css/index.css

```diff
@@ -119,6 +149,46 @@ a {
     background-color: #EE4266;
 }
 
+.murr-button__success {
+    background-color: #61E294;
+}
+
+.murr-button__success > i {
+    color: white;
+}
+
+.murr-button__success > span {
+    color: white;
+}
+
+.murr-button__success:hover {
+    background-color: #53cb89;
+}
+
+.murr-button__success:active, .murr-button__success:focus {
+    background-color: #53efa9;
+}
+
+
+.murr-button__dark {
+    background-color: #071013;
+}
+
+.murr-button__dark > i {
+    color: white;
+}
+
+.murr-button__dark > span {
```

Comment:
It's better to use classes, not tags. In this case when you change a template you don't need to fix the styles